app/src/main/res/color/item_select_color.xml

```diff
@@ -3,5 +3,5 @@
     <item android:color="?attr/colorPrimary" android:state_checked="true"/>
     <item android:color="?attr/colorPrimary" android:state_focused="true"/>
     <item android:color="?attr/colorPrimary" android:state_selected="true"/>
-    <item android:color="@color/whiteSmoke" android:state_checked="false"/>
-</selector>
\ No newline at end of file
+    <item android:color="?attr/textColor" android:state_checked="false"/>
```

Comment:
item_select_color.xml is only used within the navigation rail. This should not have any side effects